### PR TITLE
Add a sbt task to clean all projects

### PR DESCRIPTION
**Pull request by TBonnin**
## What does this change?

Create a `cleanAll` sbt task and add it to all project settings. 
This task would clean all projects, not just the currently loaded one.

## What is the value of this and can you measure success?
Happier developers 👩 👨 ➡️  😁 

To properly clean the frontend project, one had to switch to the `root` project and clean it as well as running `clean` in the current project they were working on. 
`cleanAll` is the nuclear option, cleaning all the projects (including `root`)

cc @guardian/dotcom-platform

### CODE REVIEW

**Review by jfsoul — APPROVED** [review]
Change looks good.  Can we add to the docs?

And one question: is this any different to `sbt clean` from outside?

**Comment by TBonnin** [comment]
`sbt clean` would only run `clean` on `root` which is the default project

**Comment by DiegoVazquezNanini** [comment]
This is really good! Thanks, @TBonnin! 
:v:

**Review by NataliaLKB — APPROVED** [review]
Really great!

File: docs/01-start-here/05-development-tips.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 ## Client-side development
 
+## Clean all projects
```
Comment (NataliaLKB) [inline docs/01-start-here/05-development-tips.md]: This probably shouldn't go under the `Client-side development` heading

Comment (NataliaLKB) [inline docs/01-start-here/05-development-tips.md]: Maybe under this heading instead: https://github.com/guardian/frontend/blob/78985d0cef88c1d810512241fc5243821ce49177/docs/01-start-here/05-development-tips.md#server-side-development

Comment (TBonnin) [inline docs/01-start-here/05-development-tips.md]: oups. will fix

**Pull request merged**